earthaccess/exceptions.py

```diff
@@ -0,0 +1,17 @@
+class LoginStrategyUnavailable(Exception):
+    """The selected login strategy was skipped.
+
+    This should be raised when a login strategy can't be attempted, for example because
+    "environment" was selected but the envvars are not populated.
+
+    DO NOT raise this exception when a login strategy is attempted and failed, for
+    example because credentials were rejected.
+    """
+
+    pass
+
+
+class LoginAttemptFailure(Exception):
+    """The login attempt failed."""
```

Comment:
```suggestion
    """The login attempt failed.
    
    This should be raised when a login attempt fails, for example, because
    the user's credentials were rejected.
    
    DO NOT raise this exception when a login strategy can't be attempted. For
    example, this exception would not be thrown when "environment" was selected
    but the envvars are not populated; a `LoginStrategyUnavailable` should be
    thrown instead.
    """
```

Should we be as descriptive as `LoginStrategyUnavailable`?

Reply:
:100: